src/serializer.ts

```diff
@@ -0,0 +1,115 @@
+import { ObjectSerializer } from "./api";
+import { V1ObjectMeta } from "./gen/model/v1ObjectMeta";
+
+export type AttributeType = {
```

Comment:
This is never used anywhere else, do we want to export it here for any consumer of this library to use or was this by mistake?

Reply:
It will be used when I add support for custom types.

But yes currently its not used and I will export it when it actually used